### DISCUSSION TITLE
Unexpected checkpoint *.pth error

### DISCUSSION THREAD

**Discussion by sakerlrf** [open-mmlab/mmaction2#1184] [Q&A] [answered]
Hello,

When attempting to test a model, and use a checkpoint, I get the following error:

`checkpoints/c3d_sports1m_16x1x1_45e_ucf101_rgb_20201021-26655025.pth is not a checkpoint file`

Any idea what this could be about?

Thanks a lot.

**Comment by dreamerlin** [comment, marked as answer]
Your download ckpt file is corrupted or got sth wrong, just re-download it.

And before raising an issue, I hope you can try to figure it out by yourself at first since we have provided documentation, unitest and  regression test.

**Comment by sakerlrf** [comment]
Came to the discussion, after checking the docs, and often referred to the docs in the discussion.
I hope you see the difference.

Thank you, @dreamerlin.
